### PR TITLE
reorder and partly rewrite "about" section

### PR DESCRIPTION
**Pull request by ltalirz**

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -4,45 +4,63 @@ title: About
...
-> We aim to store any kind of information correctly, reliably and in a perennial and reusable way. FAIR data is not an afterthought.
+cheminfo stands for cheminformatics tools that run in your browser.
+It includes more than 150 individual libraries and tools hosted in the following GitHub organizations:
 
-### FAIR, "machine-learning-ready" data
+- [cheminfo](https://github.com/cheminfo): parsers for various characterization techniques
+- [cheminfo-js](https://github.com/cheminfo-js): ???
+- [cheminfo-py](https://github.com/cheminfo-py): webservices (server backends) written in Python, e.g., for predicting PXRD patterns or topology analysis
+- [image-js](https://github.com/image-js): image analysis in the browser
+- [mljs](https://github.com/mljs): linear algebra and optimization in the browser
```
Comment (ltalirz) [inline docs/about.md:16]: to decide whether to keep these here?

File: docs/about.md [hunk 2]
```diff
@@ -4,45 +4,63 @@ title: About
...
+<img style="width: 30em" src="../assets/img/pxrd_overview.png">
 
-The ELN is developed in a modular fashion that makes it possible to quickly adapted to new requirements or to implement new tools or data formats. The formats that are currently supported are documented in the [database schema](https://cheminfo.github.io/data_schema/).
+Different from a paper-based lab notebook, the reagent table automatically retrieves data from large chemical databases and computes the stochiometry for you.
+For common characterization techniques in chemistry and materials science, dedicated tools are already available:
 
-### Built by chemists for chemists
+ * predict and assign NMR spectra
+ * analyze mass spectra by matching fragment
+ * or predict PXRD patterns for your crystal structures
```
Comment (ltalirz) [inline docs/about.md:37]: can we add a few more (perhaps with links?)

File: docs/about.md [hunk 3]
```diff
@@ -4,45 +4,63 @@ title: About
...
 
-The way data is entered into the ELN is similar to how you would use your paper based lab notebook. You can have a reaction diagram, a reagent table, and a description of the synthesis.
-The only difference is that everything is coupled, the reagent table automatically retrieves data from large chemical databases, and computes the stochiometry for you.
-For the most common characterization techniques in chemistry and materials science we built specific tools. You can predict and assign NMR spectra, analyze mass spectra by matching fragment or predict PXRD patterns for your crystal structures---all in the browser!
+All in the browser!
 
-<img style="width: 30em" src="../assets/img/pxrd_overview.png">
+### FAIR data, straight from the instrument
```
Comment (ltalirz) [inline docs/about.md:41]: I streamlined this section a bit.

R stands for reusable, i.e. no separate section is needed.

Also, strictly speaking, while data formats can be inherently "unfair", the format alone cannot make the data hosting FAIR. I'll add a few comments below if you'd like to reference the FAIR principles in more detail

there are no "FAIR" data formats - it's the way you host the data that

File: docs/about.md [hunk 4]
```diff
@@ -4,45 +4,63 @@ title: About
...
 
-## The cheminfo ecosystem
+> We aim to store any kind of information correctly, reliably and in a perennial and reusable way. [FAIR data](https://www.go-fair.org/fair-principles/) is not an afterthought.
+
+<img style="width: 30em" src="../assets/img/importation.png">
+
+In the ideal case shown above, data is captured automatically from the instrument, converted into an open format and ingested into the [CouchDB database](https://en.wikipedia.org/wiki/Apache_CouchDB) of the ELN.
+Where this is not possible, data can be uploaded via drag-and-drop. 
+
+All synthesis or characterization data are stored with a rich set of metadata that make them easy to *find*.
```
Comment (ltalirz) [inline docs/about.md:50]: examples of metadata might be useful.

also some note on how search works

globally unique identifiers for each data (principle F1?)?

File: docs/about.md [hunk 5]
```diff
@@ -4,45 +4,63 @@ title: About
...
+A built-in barcode system helps with building your personal chemical library and ensures that nothing is lost.
+
+Through a large library of parsers, the ELN takes care of converting all files into  standardized, open formats.
```
Comment (ltalirz) [inline docs/about.md:53]: examples of parsers (+ link?) would be useful

File: docs/about.md [hunk 6]
```diff
@@ -4,45 +4,63 @@ title: About
...
+Where this is not possible, data can be uploaded via drag-and-drop. 
+
+All synthesis or characterization data are stored with a rich set of metadata that make them easy to *find*.
+A built-in barcode system helps with building your personal chemical library and ensures that nothing is lost.
+
+Through a large library of parsers, the ELN takes care of converting all files into  standardized, open formats. 
+This is essential in order to keep information *accessible* tens of years from now -- independent of support from the instrument manufacturer.
+
+When you are ready to publish, the ELN lets you create machine-readable archives (following guidelines of the [semantic web](https://en.wikipedia.org/wiki/Semantic_Web)).
+Each archive is accompagnied by a link that your readers can used to visualize the data directly in their browser.
```
Comment (ltalirz) [inline docs/about.md:57]: maybe an example link?

File: docs/about.md [hunk 7]
```diff
@@ -4,45 +4,63 @@ title: About
...
+
+### Agile development: easy to extend
 
-We did not only build the ELN, but more than 150 other libraries which are part of the following GitHub organizations
+The cheminfo ELN is was designed as a highly modular collection of individual components.
```
Comment (ltalirz) [inline docs/about.md:61]:
```suggestion
The cheminfo ELN was designed as a highly modular collection of individual components.
```

File: docs/about.md [hunk 8]
```diff
@@ -4,45 +4,63 @@ title: About
...
+This makes it possible to quickly adapt to new requirements, to implement new tools, support new data formats, or even to implement entirely new frontend (for example, the [chemotion ELN](https://chemotion.net/) reuses some cheminfo components).
```
Comment (ltalirz) [inline docs/about.md:62]: which components are they reusing?

**Comment by ltalirz** [comment]
ready for review @kjappelbaum

File: docs/about.md [hunk 9]
```diff
@@ -4,45 +4,63 @@ title: About
...
 
-## Philosophy
+## The cheminfo ecosystem
 
-> We aim to store any kind of information correctly, reliably and in a perennial and reusable way. FAIR data is not an afterthought.
+cheminfo stands for cheminformatics tools that run in your browser.
+It includes more than 150 individual libraries and tools hosted in the following GitHub organizations:
 
-### FAIR, "machine-learning-ready" data
+- [cheminfo](https://github.com/cheminfo): parsers for various characterization techniques
```
Comment (ltalirz) [inline docs/about.md:12]: examples?

File: docs/about.md [hunk 10]
```diff
@@ -4,45 +4,63 @@ title: About
...
+- [cheminfo-js](https://github.com/cheminfo-js): ???
```
Comment (ltalirz) [inline docs/about.md:13]: how does it differ from cheminfo?

File: docs/about.md (hunk 4 above)
Comment (ltalirz) [reply docs/about.md:50]: P.S. It would be good to list the instruments here that are supported

**Comment by kjappelbaum** [comment]
> how does it differ from cheminfo? 

hehe, i think this is not a super clear cut atm

**Comment by kjappelbaum** [comment]
> which components are they reusing?
```suggestion
This makes it possible to quickly adapt to new requirements, to implement new tools, support new data formats, or even to implement entirely new frontends (for example, the [chemotion ELN](https://chemotion.net/) reuses some cheminfo components). 
``` 

good idea, added two examples which they also cite in their latest paper

**Comment by kjappelbaum** [comment]
> examples? 

added link to the app registry i'm working on

**Comment by kjappelbaum** [comment]
thanks @ltalirz, i think of the points will also be addressed once i finished https://kjappelbaum.github.io/cheminfo-app-registry/index.html (for that I want to have this other example with example files finished: https://github.com/kjappelbaum/chemical-files-registry---so users can check if their files look like that) 

thanks for your time!

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**